RELEASE.md

```diff
@@ -108,6 +109,16 @@ There may be new build versions of other dependencies (VS, MSVC).
 Our GitHub actions (tests) use the latest ones automatically,
 but the GitLab runner (release packages) doesn't.
 
+### Harden OpenSSL Defaults <a id="harden-ssl"></a>
+
+For major versions evaluate how far the global SSL defaults
```

Comment:
The protocol is called TLS since 1999, so please use that name unless there's a reason like consistency.